package.json

```diff
@@ -1,10 +1,11 @@
 {
   "name": "mx-sdk-js-bridge",
-  "version": "1.0.0",
-  "description": "",
-  "main": "index.js",
+  "version": "0.0.1",
```

Comment:
Usually, "first" usable version is `v0.1.0`.

https://stackoverflow.com/a/24154727/1475331